Source/Atomic/UI/UISelectItem.h

```diff
@@ -80,6 +80,8 @@ class UISelectItemSource : public Object
 
     void Clear() { items_.Clear(); }
 
+    const String& GetItemStr(int index);
+
```

Comment:
The /// comments are normally for doxygen and other doc processors, if you'd like that to be the standard practice, I'll make a note of that and ensure it gets done.